conf/afterinstall.sh

```diff
@@ -1 +1,5 @@
+echo "*/5 * * * * root /etc/init.d/hubble start" >> hubble-autostart
+# restart hubble once every day at 0700 hours
+echo "* 7 * * * root /etc/init.d/hubble restart" >> hubble-autostart
+mv hubble-autostart /etc/cron.d/
```

Comment:
Rather than doing this in the afterinstall.sh, can we just create the hubble-autostart file and put it in the package? If we just include it in the package then we don't have to worry about the security implications of using a tempfile in this way (and it will be removed if the package is removed -- with this current implementation, `hubble-autostart` would remain in cron.d/ which is not acceptable).